src/main/java/org/spdx/library/model/v2/license/SpdxListedLicense.java

```diff
@@ -29,6 +29,7 @@
 import org.spdx.core.DefaultModelStore;
 import org.spdx.core.IModelCopyManager;
 import org.spdx.core.InvalidSPDXAnalysisException;
+import org.spdx.core.SpdxInvalidTypeException;
```

Comment:
It looks like this import isn't really used - there is a `@throws` javadoc, but it should be changed to `InvalidSPDXAnalysisException` - the code was probably changed without changing the JavaDoc originally.  The incorrect JavaDoc shows up in a few methods in this file.

Reply:
Thank you. I will fix this

Reply:
`@throws SpdxInvalidTypeException` are now replaced with `@throws InvalidSpdxAnalysisException`